microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -753,7 +810,7 @@ To redeem your points or contribute points to a listed nonprofit organization, g
 
 #### Cancelling your participation
 
-If you no longer want to participate in the Microsoft Rewards program, follow the instructions on the [opt out page](https://account.microsoft.com/rewards/optout?confirm=false). If you opt out, you will immediately lose all of your available points. 
+If you no longer want to participate in the Microsoft Rewards program, follow the instructions on the [opt out page](https://account.microsoft.com/rewards/optout?confirm=false).<!-- todo: 404 july 10, 2024 -->  If you opt out, you will immediately lose all of your available points. 
```

Comment:
404 - update url, or comment-out sentence?
@apryzgoda 

rendered article section in PR, "opt out page" link:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3224#cancelling-your-participation
live section, "opt out page" link:
https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#cancelling-your-participation

Reply:
I logged in to my personal MS account, and found out that the URL has changed to https://rewards.bing.com/optout
Not sure if `confirm=false` is required though.

Reply:
result:
```
If you no longer want to participate in the Microsoft Rewards 
program, follow the instructions on the 
[opt out page](https://rewards.bing.com/optout).
<!-- todo: add confirm param? https://rewards.bing.com/optout?confirm=false-->
```
@captainbrosset